### DISCUSSION TITLE
Ship cmus with a .desktop file

### DISCUSSION THREAD

**Discussion by dirdi** [cmus/cmus#1096] [Integrations]
It would be convenient to ship cmus with a `cmus.desktop` file, which should be installed into `/usr/share/applications/` by `make install`. This would enable one to start cmus from application launchers like the [Xfce4 Appfinder](https://docs.xfce.org/xfce/xfce4-appfinder/start):

![2020-05-23-231240_958x1058_scrot](https://user-images.githubusercontent.com/7914030/82740623-ea2a4180-9d4a-11ea-98bc-ffb09e74d10b.png)

FWIW, that is what my `cmus.desktop` file looks like:
```
[Desktop Entry]
Version=1.0
Name=C* Music Player
Type=Application
Comment=A small, fast and powerful console music player
Comment[de]=Ein kleiner, schneller und mächtiger Musikspieler
Terminal=true
Exec=cmus
Categories=ConsoleOnly;Audio;Player;
GenericName=Music player
GenericName[de]=Musikspieler
Keywords=cmus;audio;player;
```

**Comment by Kabouik** [comment]
FWIW, here's the .desktop file I came up with too a few weeks ago. It's a bit specific, but can easily be adapted. It should:
- open `cmus` in a new Tilix terminal if not already running,
- open `cava` visualization in a new Tilix session in the same window, shown below`cmus`,
- queue double-clicked files.

```
[Desktop Entry]
Encoding=UTF-8
Version=1.0
Type=Application
Display=true
Exec=bash -c "(! pgrep cmus && tilix -e cmus && tilix -a session-add-down -e cava); sleep 0.1 && cmus-remote -q %f"
Terminal=false
Name=cmus-remote
Comment=Music player cmus-remote control
NoDisplay=true
Icon=cmus
```

**Comment by dirdi** [comment]
@Kabouik's cmus-remote approach is probably better since it enables one to not just open cmus, but also enqueue files. However, direct calling of a terminal (i.e. `tilix`) is a bad style. Instead set the `terminal` setting to `true` and the application launcher will run the command in the gobaly configured terminal.

**Comment by Kabouik** [comment]
I agree, that's not ideal. The `exec` command I use is tied to Tilix anyway since I use `-a session-add-down` to open `cava` below `cmus` in the same terminal window, but the command could be made more generic without this `cava` bit using what you suggested.
